piazza_api/rpc.py

```diff
@@ -127,6 +127,19 @@ def content_instructor_answer(self, params):
         return self._handle_error(r, "Could not create object {}.".format(
                                      repr(params)))
 
+    def content_mark_duplicate(self, params):
+        """Mark a post as duplicate to another.
+
+        :type params: dict
+        :param params: the parameters to be passed in
+        :returns python object containting returned data
+        """
+        r = self.request(
+                method="content.duplicate",
+                data=params
+        )
+        return self._handle_error(r, "Could not create object {}.".format(
+                                     repr(params)))
```

Comment:
Please follow PEP8 for your indentation; the above isn't standard. This is better:
```python
return self._handle_error(
    r,
    "Could not create object {}.".format(repr(params))
)
```